src/interactive.js

```diff
@@ -0,0 +1,32 @@
+import Utility from './utility.js';
+
+export default class Interactive {
+    static addCheckboxEvent = () => {
+      document.querySelectorAll('.checkbox').forEach((box) => box.addEventListener('change', () => {
+        const id = box.id - 1;
+        let currentStatus;
+        if (box.checked === true) {
+          currentStatus = true;
+        } else if (box.checked !== true) {
+          currentStatus = false;
+        }
+
+        this.toggleCompleted(id, currentStatus);
+      }));
+    }
+
+    static toggleCompleted = (id, currentStatus) => {
+      const todoList = Utility.getLocalStorageList();
```

Comment:
Good JavaScript actions . Its nice the way your codes are dry, there is no duplication of codes. 👍

Reply:
Thank you @Moise-code!